en/docs/edit-the-code/code-navigation.md

```diff
@@ -1 +1,31 @@
 # Code navigation
+
+## Go to definition 
+
+This feature navigates you to the definition of a particular symbol. For example, when you invoke the `Go To Definition` option on a function call expression, it navigates you to the definition of the function.
```

Comment:
`when you invoke the `Go To Definition` option on a function call expression` is this done by clicking the function call expression and selecting the **Go To Definition** from the list? 
The giff was pretty fast.

Reply:
Same as above. VSCode users are familiar with how to go to the definition, `right click -> click 'Go to Definition'` or `CTRL/CMD+Click` shortcut.

Reply:
A TW will add this by referring VS code docs